js/leaflet-sidebar.js

```diff
@@ -164,8 +170,10 @@ L.Control.Sidebar = L.Control.extend(/** @lends L.Control.Sidebar.prototype */ {
         // remove old active highlights
         for (var i = this._tabitems.length - 1; i >= 0; i--) {
             var child = this._tabitems[i];
-            if (L.DomUtil.hasClass(child, 'active'))
-                L.DomUtil.removeClass(child, 'active');
+            if (child.querySelector('a')) {
+                if (L.DomUtil.hasClass(child, 'active'))
+                    L.DomUtil.removeClass(child, 'active');
```

Comment:
is that necessary? if the code above never adds the `active` flag then there shouldn't be a need for checking it here.